### ISSUE TITLE
bug: contractOffer are not correctly mapped in edc to ids catalog

### ISSUE THREAD

**Issue by ndr-brt** [eclipse-dataspaceconnector/DataSpaceConnector#1890]
# Bug Report

## Describe the Bug
in the transformation between EDC `OfferedAsset` to IDS `Resource` there's a `_contractOffer_` field overwrite so at the end there will be only a single contract offer in the catalog pointing at the same asset:
https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/8eb991d056aa3bb6b958a751b8001578c4973238/data-protocols/ids/ids-transform-v1/src/main/java/org/eclipse/dataspaceconnector/ids/transform/type/asset/OfferedAssetToIdsResourceTransformer.java#L61-L64

_Originally posted by @ndr-brt in https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/1764#issuecomment-1228309097_

### Expected Behavior
To have a catalog with multiple contract offers

### Observed Behavior
the `_contractOffer_` field contains only the "last" offer

## Steps to Reproduce
-

## Context Information
-

## Detailed Description
-

## Possible Implementation
This should do the job:
```java
        var offers = object.getTargetingContractOffers().stream()
                .map(it -> context.transform(it, ContractOffer.class))
                .collect(Collectors.toList());
        builder._contractOffer_(offers);
```

**Comment by juliapampus** [comment]
Closed by #1891? @ndr-brt

**Comment by ndr-brt** [comment]
@juliapampus , yes, I forgot to link the issue
